### PR TITLE
Frida's Library

### PR DESCRIPTION
**Pull request by fridaforser**
## Netlify link
https://fridaslibrary.netlify.app/

### CODE REVIEW

File: index.html
Comment (NellaMori) [inline index.html:1]: I recommend using a formatter to have the correct spacing and indentation, also change the tab size settings from 4 to 2 please 😄 you can see now there are a lot of empty spaces in the code that compromise the readability 👍

File: style.css
Comment (NellaMori) [inline style.css:1]: same here with formatting tabs number from 4 -> 2 🧹

File: style.css [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    border: none;
+    margin-top: -40px;
+    font-family: 'Kumbh Sans';
+    font-weight: bold;
+    font-size: 11px;
+    margin-bottom: 13px;
+    box-shadow: 2px 2px 2px #00a2ff33;
+}
+
+button:hover {
```
Comment (NellaMori) [inline style.css:116]: When you apply an effect to the button, like in this case the transition, is making all the UI around move, and this is to be avoided. A solution would be to wrap the button in a div and give the div a fixed height 👍

**Review by NellaMori — APPROVED** [review]
Hi Frida, congrats you did a really good job with this project 🔥 You followed the instructions meticulously, implemented basic requirements + stretch goals and even followed the design to inspire the UI of your app. Overall a very good execution considering you also worked on your own 👍 I left some comments around the code for little improvements, keep up the fantastic work! 🏅

**Review by AHPIXI — COMMENTED** [review]
Me and Lovisa has reviewed your code. We both think it's great. We have tested your code and everything works as it should. It also looks very nice.
The code is written in a clean and clear manner. The code fulfil the requirements of the assignment. Well done! 🌟

**Pull request closed without merging**